Ladybird/.clang-format

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
no longer needed, should match top-level clang-format config exactly

Reply:
One diff, but I don't think we have any(?) concepts use in Ladybird application code 
```
diff .clang-format Ladybird/.clang-format 
15,16d14
< RequiresClausePosition: WithFollowing
< IndentRequiresClause: false
```

Reply:
Removed with `git filter-branch -f --index-filter 'git rm --cached --ignore-unmatch Ladybird/.clang-format' master..HEAD`